heron/proto/ckptmgr.proto

```diff
@@ -112,8 +112,15 @@ message RestoreTopologyStateResponse {
 // to some valid checkpoint. This is sent either if stmgr dies
 // and comes back up or if an instance dies.
 message ResetTopologyState {
+  // Was there a dead/recovered stmgr connection that was the reason
+  // for this request?
   optional string dead_stmgr = 1;
+  // Was there a dead/recovered local instance connection that was the reason
+  // for this request?
   optional int32 dead_taskid = 2;
```

Comment:
Not necessary. We issue these messages to the tmaster as soon as we notice any instance failure. The same applies for any stmgr client connection failures